examples/server_async.py

```diff
@@ -167,7 +166,6 @@ async def run_async_server(args):
             # custom_functions=[],  # allow custom handling
             framer=args.framer,  # The framer strategy to use
             # handler=None,  # handler for each session
-            # TBD allow_reuse_address=True,  # allow the reuse of an address
```

Comment:
Needed for TCP.

And all similar places, not UDP (I will not comment on the rest).